Gemfile

```diff
@@ -17,14 +17,15 @@ group :assets do
 end
 
 group :mongoid do
-  gem 'mongoid'
+  gem 'mongoid', github: 'mongodb/mongoid', ref: 'a79e90cb67b8a5865e99a1c6d7cb11e0d7591e7c'
```

Comment:
👍 
can we add a note here with context like the `apparition` gem below does?

also, might not be a bad idea to add this to our clean up card in jira (in case they come out with a proper release by the time we're finished)